planning/behavior_path_planner/autoware_behavior_path_lane_change_module/src/manager.cpp

```diff
@@ -330,25 +332,16 @@ void LaneChangeModuleManager::updateModuleParams(const std::vector<rclcpp::Param
       parameters, ns + "min_prepare_duration", p->trajectory.min_prepare_duration);
     updateParam<double>(parameters, ns + "lateral_jerk", p->trajectory.lateral_jerk);
     updateParam<double>(
-      parameters, ns + ".min_lane_changing_velocity", p->trajectory.min_lane_changing_velocity);
-    // longitudinal acceleration
+      parameters, ns + "min_lane_changing_velocity", p->trajectory.min_lane_changing_velocity);
     updateParam<double>(
       parameters, ns + "min_longitudinal_acc", p->trajectory.min_longitudinal_acc);
     updateParam<double>(
       parameters, ns + "max_longitudinal_acc", p->trajectory.max_longitudinal_acc);
     updateParam<double>(
       parameters, ns + "lane_changing_decel_factor", p->trajectory.lane_changing_decel_factor);
     int longitudinal_acc_sampling_num = 0;
-    updateParam<int>(parameters, ns + "lon_acc_sampling_num", longitudinal_acc_sampling_num);
-    if (longitudinal_acc_sampling_num > 0) {
-      p->trajectory.lon_acc_sampling_num = longitudinal_acc_sampling_num;
-    }
-
-    int lateral_acc_sampling_num = 0;
-    updateParam<int>(parameters, ns + "lat_acc_sampling_num", lateral_acc_sampling_num);
-    if (lateral_acc_sampling_num > 0) {
-      p->trajectory.lat_acc_sampling_num = lateral_acc_sampling_num;
-    }
+    updateParam<int>(parameters, ns + "lon_acc_sampling_num", p->trajectory.lon_acc_sampling_num);
```

Comment:
these acts as a guard in case someone accidentally set and invalid values.

Reply:
@zulfaqar-azmi-t4 
I added warning messages in https://github.com/autowarefoundation/autoware.universe/pull/9887/commits/2b1decb4a36fefa2b9b91d75207380bfcfd8a86a using `RCLCPP_WARN` because in the original implementation, users wouldn't be able to notice when parameters are not being updated.
Can you review?